src/pages/index.jsx

```diff
@@ -16,6 +16,13 @@ class IndexPage extends Component {
     this.instance.submitCallback = function someFancyfunction(data) {
       var stringified = JSON.stringify(data);
       var testVariable = 'This is what we have got after submit' + stringified;
+      console.error(testVariable);
+    };
+    this.instance.successCallback = function() {
```

Comment:
maybe we could replace those alerts with console messages? :)

Reply:
sure, done :)